index.html

```diff
@@ -50,18 +50,31 @@ <h2>Recruiting industry leaders to serve as change agents</h2>
             </div>
           </div>
           <div class="row" id="map-data">
+            <!-- Updating this section Step 1:
+              ... to see the most recent site data counts (including agencies, fellows, fellows by region, etc.),
+              ... run `JEKYLL_ENV=dev_logging bundle exec jekyll build` from the command line
+            -->
+            <!-- Updating this section Step 2:
+              ... in the PIF shared google drive under Design > Visual Design Assets,
+              ... find the map source file named something like "PIF-regions-map-YYYYMMDD.key",
+              ... then open the file, and edit each of the numbers,
+              ... and recognize each of the circles is really a pie chart,
+              ... and double-click on each of the pie charts to "Edit Chart Data",
+              ... then enter in the updated total number of PIFs, and the updated number of PIFs for that region,
+              ... then save a new version of the keynote file to google drive,
+              ... then take a screenshot of the final map and save to this repo as "assets/images/map/pif-map-YYYYMMDD.png",
```

Comment:
It could help to clarify to what repo you refer by "this repo"

Reply:
Although I guess you specify the file name requested right after, so you can probably ignore my comment.